package.json

```diff
@@ -49,6 +49,7 @@
     "gatsby-transformer-remark": "^1.7.2",
     "gatsby-transformer-sharp": "^1.6.1",
     "glamor": "^2.20.40",
+    "graphql": "0.10.5",
```

Comment:
I wouldn't expect to see this change, or any of the changes to the `yarn.lock` file below, given this PR mostly touches markdown files. Did you mean to commit them?